eng/submit-source-build-release-pr.sh

```diff
@@ -21,13 +21,15 @@ a PR in the target repo."
    echo "--targetBranch, -b    (Optional) branch to submit the PR to, calculated automatically otherwise"
    echo "--globalJson, -g      (Optional) path to the global.json file to update"
    echo "--versionsProps, -p   (Optional) path to the Versions.props file to update"
+   echo "--resolvedSourceBuiltArtifactsFileName, -a   (Optional) actul Source Built Artifacts File Name"
```

Comment:
The value is being passed to PrivateSourceBuiltArtifactsUrl, would it be better to name it accordingly? The same for PrivateSourceBuiltSdkUrl_CentOS8Stream. Also, what can be passed into these variables? Is it URL, file paths or eventually globs? Just checking if there is a more fitting name.

Reply:
The variable was initially intended to have data for all versions. Prior versions used PrivateSourceBuiltArtifactsPackageVersion; however, in Net8, PrivateSourceBuiltArtifactsUrl is used instead. I continued to use the same name despite the updated code. Thanks for the reminder. Done, Thank you :)